etcd/application/certs.yaml

```diff
@@ -0,0 +1,147 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: etcd-certs
+stringData:
```

Comment:
Instead of duplicating the other files, should you use a secret generator from the kustomization to load these all in.